Cargo.toml

```diff
@@ -42,8 +42,8 @@ derive_builder = "0.13.0"
 either = "1"
 env_logger = "0.11.0"
 futures = "0.3"
-iceberg = { path = "./crates/iceberg" }
-iceberg-catalog-rest = { path = "./crates/catalog/rest" }
+iceberg = { version = "0.2.0", path = "./crates/iceberg" }
+iceberg-catalog-rest = { version = "0.2.0", path = "./crates/catalog/rest" }
```

Comment:
This is in fact not relied by anyone, consider removing this for now?

Reply:
Used by examples.

Reply:
```shell
error: failed to load manifest for workspace member `/home/xuanwo/Code/apache/iceberg-rust/crates/examples`

Caused by:
  failed to parse manifest at `/home/xuanwo/Code/apache/iceberg-rust/crates/examples/Cargo.toml`

Caused by:
  error inheriting `iceberg-catalog-rest` from workspace root manifest's `workspace.dependencies.iceberg-catalog-rest`

Caused by:
  `dependency.iceberg-catalog-rest` was not found in `workspace.dependencies`

```